run.go

```diff
@@ -57,6 +57,10 @@ command(s) that get executed on start, edit the args parameter of the spec. See
 			Name:  "no-new-keyring",
 			Usage: "do not create a new session keyring for the container.  This will cause the container to inherit the calling processes session key",
 		},
+		cli.IntFlag{
+			Name:  "preserve-fds",
+			Usage: "Pass N additional file descriptors to the container (stdio + $LISTEN_FDS + N in total)",
```

Comment:
If we're going to have this sort of API, I would prefer that we make it more explicit than this. For example, make it so that you have to specify **which** file descriptors you want to pass (`IntSliceFlag`). This would also mean that cases where you don't want to use low file descriptors (see [this SQLite option for more reasons why that might matter](https://www.sqlite.org/compile.html#minimum_file_descriptor)).